node/cn/backend.go

```diff
@@ -569,7 +569,10 @@ func (s *CN) SetupKaiaxModules(ctx *node.ServiceContext, mValset valset.ValsetMo
 	// Register modules to respective components
 	// TODO-kaiax: Organize below lines.
 	s.RegisterBaseModules(s.stakingModule, mReward, mSupply, s.govModule, mValset, mRandao)
-	s.RegisterJsonRpcModules(s.stakingModule, mReward, mSupply, s.govModule, mRandao, mBuilder, mGasless)
+	if !gaslessDisabled {
+		s.RegisterJsonRpcModules(s.stakingModule, mReward, mSupply, s.govModule, mRandao, mBuilder, mGasless)
+	}
+	s.RegisterJsonRpcModules(s.stakingModule, mReward, mSupply, s.govModule, mRandao, mBuilder)
```

Comment:
This is run even if gasless is disabled. How about use the variable []kaiax.JsonRpcModule like mExecution?

```golang
mJsonRpc := []kaiax.JsonRpcModule{}
```

Reply:
like below?

```
	mJsonRpc := []kaiax.JsonRpcModule{s.stakingModule, mReward, mSupply, s.govModule, mRandao, mBuilder}

	gaslessDisabled := mGasless.IsDisabled()
	if !gaslessDisabled {
		mExecution = append(mExecution, mGasless)
		mTxBundling = append(mTxBundling, mGasless)
		mTxPool = append(mTxPool, mGasless)
		mJsonRpc = append(mJsonRpc, mGasless)
	}
	s.RegisterJsonRpcModules(mJsonRpc...)
```

Reply:
yes